### ISSUE TITLE
[VT-4] vit; new keybind token; # represents current filter

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#26]
_David Patrick on 2013-09-08T19:29:49Z says:_ 

 So that something like 

   map R=trev.pl # list

would act on the current filter set.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-09-08T19:29:49Z
Modified: 2014-02-09T01:14:58Z
```

**Comment by taskwarrior** [comment]
_David Patrick on 2013-09-08T19:42:18Z says:_ 

 or maybe this would be better as an expansion of the % token behavior.

If a task is selected then % represents that task.
If multiples are selected, (see #1155) % represents _those_ tasks.
If NO tasks are selected (see #1368) then all the tasks in the current report (a.k.a. the current filter) are (is) represented.

This would be better/ more flexible.

THAT SAID, implementing # to simply mean < the filter set > is probably much easier than the above suggestion requiring #1155 and #1368, which are doozies. 
"#" as filter might be a good short-term solution with a selection-sensitive-% as future target.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-09-15T06:35:02Z says:_ 

 Fixed at 1da206b8.
